code/game/objects/items/weapons/grenades/custom_grenades.dm

```diff
@@ -6,12 +6,16 @@
 /obj/item/grenade/chem_grenade/dirt/Initialize(mapload)
 	. = ..()
 	var/obj/item/reagent_containers/glass/beaker/B1 = new(src)
-	var/list/muck = list("blood","carbon","flour","radium")
-	var/filth = pick(muck - "radium") // not usually radioactive
+	var/list/muck = list("blood","carbon","flour")
+	var/filth = pick(muck)
+	muck -= filth
 
-	B1.reagents.add_reagent(filth,25)
+	B1.reagents.add_reagent(filth, 25)
 	if(prob(25))
-		B1.reagents.add_reagent(pick(muck - filth,25)) // but sometimes...
+		if(prob(25))
```

Comment:
Is there a reason for the double prob(25) as opposed to a single probability?

Reply:
I realized theres a better way I can do this now with just adding radium to muck and reducing the probs. I originally had them like this because I couldn't think of a better method without calling something like prob(6.25) and then prob(18.75), which looks even worse. Updating with next commit